mypy/typeops.py

```diff
@@ -5,7 +5,8 @@
       since these may assume that MROs are ready.
 """
 from collections import defaultdict
-from typing import cast, Optional, List, Sequence, Set, Iterable, TypeVar, Dict, Tuple, Any
+from typing import cast, Optional, List, Sequence, Set, Iterable, TypeVar, Dict, Tuple, Any, \
```

Comment:
whats the changes in this file for? doesnt seem to be related to the tests

Reply:
So running the legacy tests in based mode led to some crashes, these changes are to fix that. I think the only change in this file is to correctly collect the return type from an `async def` overload.